README.md

```diff
@@ -148,10 +148,15 @@ When adding new .env vars do not forget to update the [publish.yml](.github/work
 
 0. add a test for determining the correct issuance module [here](./src/utils/issuanceModules.test.ts)
 1. add a test for determining the correct contract [here](./src/utils/contracts.test.ts)
-2. if there is a new FlashMint contract, add it [here](./src/constants/contracts.ts)
-3. additionally, for new contracts add a new [builder](./src/flashmint/builders/) and [quote provider](./src/quote/)
-4. a new quote provider has to be integrated in the [FlashMintQuoteProvider](./src/quote/indexQuoteProvider.ts)
-5. ...
+2. if there is a new FlashMint contract, add it as described [below](#adding-a-new-contract)
+3. additionally, add a test in [tests](./src/tests/)
+
+### Adding a new contract
+
+0. add the contract address in [constants](./src/constants/contracts.ts)
+1. add appropriate getters and tests in [utils/contracts](./src/utils/contracts.ts)
+2. add a new [builder](./src/flashmint/builders/) and [quote provider](./src/quote/)
+3. The new quote provider has to be integrated into the [FlashMintQuoteProvider](./src/quote/indexQuoteProvider.ts)
```

Comment:
Added a README so we have some steps what files to check when adding a new contract.